build-support/travis/travis.yml.mustache

```diff
@@ -660,32 +657,25 @@ matrix:
       script:
         - ./build-support/bin/travis-ci.sh -2n
 
-    # TODO: get this passing with a Python 3 PEX.
-    - <<: *py2_linux_test_config
-      name: "Python contrib tests (Py2 PEX w/ Py3 constraints)"
-      stage: *test
+    - <<: *py3_linux_test_config
+      name: "Python contrib tests (Py3 PEX)"
       env:
-        - *py2_linux_test_config_env
-        - PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS="['CPython>=3.6']"
+        - *py3_linux_test_config_env
         - CACHE_NAME=linuxcontribtests.py3
       script:
-        - ./build-support/bin/travis-ci.sh -2n
+        - ./build-support/bin/travis-ci.sh -n
 
     - <<: *py2_osx_10_12_sanity_check
     - <<: *py3_osx_10_12_sanity_check
 
     - <<: *py2_osx_10_13_sanity_check
     - <<: *py3_osx_10_13_sanity_check
 
-    # TODO: get this to pass using Python 3. In the meantime, just use Python 2.
     - <<: *py2_osx_platform_tests
-      stage: *test
-    # - <<: *py3_osx_platform_tests
+    - <<: *py3_osx_platform_tests
 
-    # TODO: get this to pass using Python 3. In the meantime, just use Python 2.
     - <<: *py2_jvm_tests
-      stage: *test
-    # - <<: *py3_jvm_tests
+    - <<: *py3_jvm_tests
```

Comment:
In general, it's difficult for me to follow having lots of `.travis.yml` changes at the same time as code changes without some explanation of why we are now allowed to do this because of the code changes. Issue numbers especially are deeply appreciated on almost any `TODO`, as well as linked in the ticket description.